DataGrid: Add External Pager Info to Paging Demo Desc. (#32920)
DevExpress/DevExtreme: apps/demos/Demos/DataGrid/RecordPaging/description.md

> The DevExtreme DataGrid supports data paging. The component can load rows in portions to improve performance when working with large data sets. This demo allows you to navigate between pages using the DataGrid's built-in pager.

Reviewer: Well, technically we know that we added "built-in" to make a contrast with "custom". But the reader doesn't. Since I am edititing this part anyway, I'll remove it here to avoid repetitions.

```suggestion
The DevExtreme DataGrid supports data paging. The component can load rows in chunks to improve performance when working with large data sets. This demo allows you to navigate between pages using the DataGrid pager.
```

> You can use controls below the DataGrid to change the pager display mode and toggle the visibility of individual pager elements. Note: in compact mode, navigation buttons are always visible.

Reviewer: ```suggestion
You can use controls below the DataGrid to change pager display mode and toggle the visibility of individual pager elements. Note: navigation buttons are always visible in compact mode.
```

The built-in [pager](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/) contains the following elements:

- **Page navigator**        
Enables page navigation.

- **Page size selector**        

> Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showPageSizeSelector) property. You can also define [allowed page sizes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify an initial [page size](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#pageSize).

Reviewer: "the" was fine

```suggestion
Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showPageSizeSelector) property. You can also define [allowed page sizes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [page size](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#pageSize).
```

- **Page information**           
Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showInfo) property. You can customize this [information text](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#infoText) as needed.

The built-in pager supports full, compact, and adaptive (default) [display modes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#displayMode). In compact mode, the pager uses less screen space. In adaptive mode, the DataGrid automatically chooses between full and compact modes based on the component width.

> In this demo, you can use controls below the DataGrid to change the pager display mode and toggle the visibility of individual pager elements. Note that in compact mode, navigation buttons are always visible.

Reviewer: We can move this sentence before split.

```suggestion
In this demo, you can use controls below the DataGrid to change the pager display mode and toggle the visibility of individual pager elements. Note: in compact mode, navigation buttons are always visible.
```

> The DevExtreme DataGrid also supports external pagers. You can hide the built-in pager and configure a standalone [Pagination](/Documentation/Guide/UI_Components/Pagination/Overview/) component to control the DataGrid. For additional information, refer to the following example: [DevExtreme DataGrid - Display a Pager Above the Grid](https://github.com/DevExpress-Examples/devextreme-datagrid-pager-on-top).

Reviewer: ```suggestion
The DevExtreme DataGrid also supports external pagers. You can hide the built-in pager and configure a standalone [Pagination](/Documentation/Guide/UI_Components/Pagination/Overview/) component to navigate the DataGrid. For additional information, refer to the following example: [DevExtreme DataGrid - Display a Pager Above the Grid](https://github.com/DevExpress-Examples/devextreme-datagrid-pager-on-top).
```

The DataGrid splits records into multiple pages. This technique helps optimize control performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the built-in pager.
<!--split-->

> The built-in pager is configured in the [pager](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/) object and contains the following elements:

Reviewer: We can get rid of passive here
